rust/main/agents/scraper/src/agent.rs

```diff
@@ -190,8 +190,9 @@ impl Scraper {
 
         Ok(tokio::spawn(
             async move {
-                // If any of the tasks panic, we want to propagate it, so we unwrap
-                try_join_all(tasks).await.unwrap();
+                if let Err(err) = try_join_all(tasks).await {
+                    tracing::error!(?err, "Some scraper tasks failed");
+                }
```

Comment:
Can we replace `unwrap` with `expect` since tokio will capture the panic and return an error.

Reply:
sure 👍